wails-frontend/src/pages/backupKeyPairs.tsx

```diff
@@ -0,0 +1,116 @@
+import { useState } from 'react';
+import { promptRequest } from '../events/events';
+import { useLocation } from 'react-router-dom';
+import { handleCancel } from '../utils/utils';
+import { Password, Button } from '@massalabs/react-ui-kit';
+import { FiCopy } from 'react-icons/fi';
+
+function BackupKeyPairs() {
+  const [inputPassword, setInputPassword] = useState<string | undefined>(
+    undefined,
+  );
+  const [show, setShow] = useState<boolean>(false);
+  const [privateKey, setPkey] = useState<string | undefined>(undefined);
+
+  let { state } = useLocation();
+  const req: promptRequest = state.req;
+
+  const baselineStr = 'Enter your password to show your private key';
+
+  function handleCopy() {
+    console.log('copy');
```

Comment:
🤔

Reply:
![image](https://github.com/massalabs/thyra-plugin-wallet/assets/72019005/c8889c8b-ad17-4bf2-90e1-3dc603b84a39)

@mazmassa I think @pivilartisant has implemented this function to inform the backend.